.github/workflows/test.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        suite: [ repo, fedprops, repo-client ]
+        suite: [ repo, fedprops ]
```

Comment:
Is this supposed to be included in this commit?

Reply:
no that looks wrong